hummingbot/connector/exchange/ndax/ndax_user_stream_tracker.py

```diff
@@ -27,16 +25,22 @@ def logger(cls) -> HummingbotLogger:
         return cls._logger
 
     def __init__(
-        self, throttler: AsyncThrottler, shared_client: Optional[aiohttp.ClientSession] = None, auth_assistant: Optional[NdaxAuth] = None, domain: Optional[str] = None
+            self,
+            throttler: AsyncThrottler,
+            shared_client: Optional[aiohttp.ClientSession] = None,
+            auth_assistant: Optional[NdaxAuth] = None,
+            domain: Optional[str] = None
     ):
-        super().__init__()
         self._auth_assistant: NdaxAuth = auth_assistant
         self._shared_client = shared_client
-        self._ev_loop: asyncio.events.AbstractEventLoop = asyncio.get_event_loop()
-        self._data_source: Optional[UserStreamTrackerDataSource] = None
-        self._user_stream_tracking_task: Optional[asyncio.Task] = None
         self._domain = domain
         self._throttler = throttler
+        super().__init__(data_source=NdaxAPIUserStreamDataSource(
```

Comment:
As discussed in the last stand up meeting, we need to do this in this case to instantiate the data source required by super().__init__. The class should be removed in future cleanups anyways, so we are accepting this situation for the moment, to avoid introducing big changes in all connectors with this PR intended for Kucoin.